robotiq_c_model_control/src/robotiq_c_model_control/c_model_ethercat_client.cpp

```diff
@@ -0,0 +1,77 @@
+#include "robotiq_c_model_control/c_model_ethercat_client.h"
+
+#include "robotiq_ethercat/ethercat_manager.h"
+
+// See Robotiq's documentation for the register mapping
+
+// An effort to keep the lines less than 100 char long
+namespace c_model = robotiq_c_model_control;
```

Comment:
I prefer to explicitly qualify all the names in source files, but I admit its really ugly to have to remap namespaces. I'll change this.